pkgs/by-name/ni/nixbit/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  ninja,
+  pkg-config,
+  kdePackages,
+  qt6,
+  libgit2,
+  installShellFiles,
+  xvfb-run,
+  fetchFromGitHub,
```

Comment:
ordering of these libs could be a bit better, I tend to prefer to have the bare minimal stuff like lib, stdenv, git etc at the top, then the deps later.